components/About/AboutDetails.styled.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from "styled-components";
+
+export const ContentContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  padding: 42px;
+  overflow-x: hidden;
+
+  @media screen and (max-width: 768px) {
+    padding: 16px;
+  }
+`;
+
+export const AboutText = styled.p`
+  font-size: 16px;
+  margin-bottom: 10px;
+`;
+
+export const Phone = styled.p`
+  font-size: 16px;
```

Comment:
since phone, about and address have the CSS rules, you could create a `Text` component that accepts the margin as a optional prop